### DISCUSSION TITLE
How can I configure the legend font size?

### DISCUSSION THREAD

**Discussion by nathan-at-least** [plotters-rs/plotters#397] [Q&A]
New to plotters, and still trying to piece together the extensive API.

I've increased the render size on a png and then found all the fonts were too small to see. My basic line plot has text in the caption, the axes, and the legend. I've found how to configure the first two:

- `ChartBuilder::caption(… (fontname, size))`
- `ChartContext::configure_mesh()` -> `MeshStyle::label_style((fontname, size))`

I can't figure out how to configure the legend font size. The call site looks like:

```
for (ix, dset) in datasets.into_iter().enumerate() {
    let color = PALETTE[ix % PALETTE.len()];
    chart
        .draw_series(LineSeries::new(dset.points.into_iter(), color))?
        .label(dset.name)
        .legend(move |(x, y)| PathElement::new(vec![(x, y), (x + 20, y)], color));
}   
```

I see `SeriesAnno::label` but nothing that takes an `IntoTextStyle`.

**Comment by RiwEZ** [comment]
Use `configure_series_labels()` by doing something like below code.
```rust
chart.configure_series_labels().label_font(("Hack", 14).into_font())
```
